patterns/templates/shared/ready.js

```diff
@@ -4,12 +4,12 @@
 *
```

Comment:
this shared folder is actually not a template, can this be moved?

Reply:
moved